packages/accounts/src/msca/multi-owner-account.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  SignerSchema,
+  createBaseSmartAccountParamsSchema,
+  type SignTypedDataParams,
+  type SupportedTransports,
+} from "@alchemy/aa-core";
+import {
+  concatHex,
+  encodeFunctionData,
+  hashMessage,
+  hashTypedData,
+  hexToBytes,
+  isBytes,
+  toBytes,
+  type FallbackTransport,
+  type Hash,
+  type Transport,
+} from "viem";
+import { z } from "zod";
+import { MultiOwnerMSCAFactoryAbi } from "./abis/MultiOwnerMSCAFactory.js";
+import { MSCABuilder, StandardExecutor } from "./builder.js";
+import { MultiOwnerPlugin } from "./plugins/multi-owner.js";
+
+export const createMultiOwnerMSCASchema = <
+  TTransport extends SupportedTransports = Transport
+>() =>
+  createBaseSmartAccountParamsSchema<TTransport>().extend({
+    owner: SignerSchema,
+    index: z.bigint().optional().default(0n),
```

Comment:
what does this `index` specify?

Reply:
index is used as a salt, if you want to create many accounts with the same set of initial owners but have them be at other addresses use can use the salt to differentiate them